codespell_lib/data/dictionary.txt

```diff
@@ -5901,6 +5928,7 @@ footnoes->footnotes
 foppy->floppy
 foramatting->formatting
 foramt->format
+forat->format
```

Comment:
Add fort?

Reply:
Please.

Reply:
@Gelma I think this got missed.

Reply:
@Gelma I don't know what's going on here, but this is no longer in the PRs changes, unless it got merged elsewhere. Anyway please can you add `forat->format, fort`

Reply:
@peternewman: well, it's also in "Misc queue from 2018 (sorry, lost sources) (#895)".

It seems already merged to me.

Reply:
Anyway, before to submit patches, I have a big file with all my entries. So in the end I will be able to check if something got missing.